docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+      \\
+      \mathbb{d}_1 = \xi_{2, 0, \ldots, 0} (i,j), &
+      \mathbb{d}_2 = \xi_{0,2, \ldots, 0} (i,j), &
+      \ldots, &
+      \mathbb{d}_n = \xi_{0,0, \ldots, 2} (i,j), \\
+      \\
+      \mathbb{f}_1 = \xi_{3, 0, \ldots, 0} (i,j), &
+      \mathbb{f}_2 = \xi_{0,3, \ldots, 0} (i,j), &
+      \ldots, &
+      \mathbb{f}_n = \xi_{0,0, \ldots, 3} (i,j), \\
+      \vdots & \vdots &  & \vdots
+      \end{array}
+
+    For weakly coupled homonuclear spins it is also convenient to define
+
+    .. math::
+
+      \begin{array}{c}
+      \mathbb{p}_{1,2,\ldots,n} =  \mathbb{p}_{1}
+      + \mathbb{p}_{2} + \cdots \mathbb{p}_{n}, \\
+      \\
+      \mathbb{d}_{1,2,\ldots,n} =  \mathbb{d}_{1}
+      + \mathbb{d}_{2} + \cdots \mathbb{d}_{n}, \\
+      \\
+      \mathbb{f}_{1,2,\ldots,n} =  \mathbb{f}_{1}
+      + \mathbb{f}_{2} + \cdots \mathbb{f}_{n}, \\
+      \vdots
+      \end{array}
+
+
+    When the :math:`\ell` are zero on all nuclei except two, then we identify
+    these functions using a concatenation of sub-shell letter designations, e.g.,
+
+    .. math::
+
+      \begin{array}{cccc}
+      (\mathbb{pp})_{1,2} = \xi_{1,1,0, \ldots, 0} (i,j), &
+      (\mathbb{pp})_{1,3} = \xi_{1,0,1, \ldots, 0} (i,j), &
+      \ldots, &
+      (\mathbb{pp})_{1,n} = \xi_{1,0,0, \ldots, 1} (i,j),\\
+      \\
+      (\mathbb{pd})_{1,2} = \xi_{1, 2, 0, \ldots, 0} (i,j), &
+      (\mathbb{pd})_{1,3} = \xi_{1,0,2 \ldots, 0} (i,j), &
+      \ldots, &
+      (\mathbb{pd})_{1,n} = \xi_{1,0, \ldots, 2} (i,j), \\
+      \\
+      (\mathbb{dp})_{1,2} = \xi_{2, 1, 0, \ldots, 0} (i,j), &
+      (\mathbb{dp})_{1,3} = \xi_{2 ,0, 1 \ldots, 0} (i,j), &
+      \ldots, &
+      (\mathbb{dp})_{1,n} = \xi_{2, 0, \ldots, 1} (i,j), \\
+      \vdots & \vdots &  & \vdots
+      \end{array}
+
+    Below is an energy level diagram of two coupled spin :math:`I=1/2` nuclei with
+    transition labeled according to their transition symmetry function values.  Note
+    that each transition has a unique set of transition symmetry function values.
+
+    .. figure:: ../../_static/CoupledOneHalf.*
+        :width: 650
+        :alt: figure
+        :align: center
+
+
+Single-Spin Queries
+-------------------
+
+Based on the review above, we now know that the spin :math:`I=1`, the transition
+:math:`\ketbra{-1}{0}` can be selected with :math:`{(\text{p}_I,\text{d}_I) =
+(-1,1)}`.  In **mrsimulator**, this transition is selected during a
+**SpectralEvent** using the **SymmetryQuery** and **TransitionQuery** objects,
+as defined in the code below.
+
+.. plot::
+    :context: reset
+
+    from mrsimulator.method.query import SymmetryQuery, TransitionQuery
+    from mrsimulator.method import SpectralEvent
+
+    symm_query = SymmetryQuery(P=[-1], D=[1])
+    trans_query = TransitionQuery(ch1=symm_query)
+    event = SpectralEvent(fraction=1, transition_query=[trans_query])
+
+.. note::
+    Python dictionaries can also be used to create and initialize an object. To do this,
```

Comment:
> and initialize an object

and initialize **mrsimulator** objects.

Reply:
that's a question I had.  Is it only mrsimulator objects, or is this a general thing in Python?